lib/routes/account.js

```diff
@@ -425,6 +432,15 @@ module.exports = function (
             .then(
               function (result) {
                 device = result
+                if (operation === 'createDevice') {
+                  log.event('device', {
+                    action: 'create',
```

Comment:
I wonder whether separate `createDevice` and `deleteDevice` events would be simpler than having `device` event with an `action` property...@dannycoates any preference?